.devcontainer/devcontainer.json

```
@@ -2,35 +2,37 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/v0.194.3/containers/dapr-javascript-node
 {
 	"name": "Vehicle Model Generator",
-	"build": {
-		"dockerfile": "Dockerfile"
-	},
-	"runArgs": [
-		"--init",
-		"--privileged"
-	],
-	"settings": {
-		"python.pythonPath": "/usr/bin/python3",
-		"python.defaultInterpreterPath": "/usr/bin/python3",
-		"python.linting.enabled": true,
-		"python.formatting.provider": "black",
-		"python.linting.banditEnabled": true,
-		"python.linting.flake8Enabled": true,
-		"python.disableInstallationCheck": true,
-		"terminal.integrated.defaultProfile.linux": "zsh",
-		"terminal.integrated.profiles.linux": {
-			"zsh": {
-				"path": "/usr/bin/zsh"
-			}
+    "image": "ghcr.io/eclipse-velocitas/devcontainer-base-images/python:v0.2",
+	"customizations": {
+		"vscode": {
+			"settings": {
+				"python.pythonPath": "/usr/bin/python3",
+				"python.defaultInterpreterPath": "/usr/bin/python3",
+				"python.linting.enabled": true,
+				"python.formatting.provider": "ruff",
+				"python.disableInstallationCheck": true,
+				"terminal.integrated.defaultProfile.linux": "zsh",
+				"terminal.integrated.profiles.linux": {
+					"zsh": {
+						"path": "/usr/bin/zsh"
+					}
+				}
+			},
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"ms-python.python",
+				"dotjoshjohnson.xml",
+				"matangover.mypy",
+				"the-compiler.python-tox",
+				"Semgrep.semgrep",
+				"charliermarsh.ruff",
+				"donjayamanne.python-environment-manager",
+				"njpwerner.autodocstring",
+				"donjayamanne.python-extension-pack",
+				"tamasfe.even-better-toml"
+			]
```

whats about add ms-python.mypy-type-checker?
```suggestion
			"extensions": [
				"ms-python.python",
				"dotjoshjohnson.xml",
				"matangover.mypy",
				"the-compiler.python-tox",
				"Semgrep.semgrep",
				"charliermarsh.ruff",
				"donjayamanne.python-environment-manager",
				"njpwerner.autodocstring",
				"donjayamanne.python-extension-pack",
				"tamasfe.even-better-toml",
				"ms-python.mypy-type-checker"
			]
```